addons/account/models/chart_template.py

```diff
@@ -184,7 +189,25 @@ def try_loading(self, company=False):
         if not company.chart_template_id and not self.existing_accounting(company):
             for template in self:
                 template.with_context(default_company_id=company.id)._load(15.0, 15.0, company)
-
+            # Install the demo data when the first localization is instanciated on the company
+            if install_demo and self.env.ref('base.module_account').demo:
+                self.with_context(
+                    default_company_id=company.id,
+                    allowed_company_ids=[company.id],
+                )._create_demo_data()
+
+    def _create_demo_data(self):
+        try:
+            demo_data = self._get_demo_data()
+            for model, record in demo_data:
+                created = self.env[model]._load_records([{
+                    'xml_id': "account.%s" % xml_id if '.' not in xml_id else xml_id,
+                    'values': data,
+                } for xml_id, data in record])
+                self._post_create_demo_data(created)
+        except Exception:
```

Comment:
can you be a bit more specific on what could go wrong in the caught execption? I understand you don't want to prevent installation because on demo data so maybe at least log the error content. The `try/except` should maybe be set around the `_create_demo_data` call, in `try_loading` if this is the scenario you don't want to be interupted in.

Reply:
> so maybe at least log the error content

That is what `_logger.exception` does